### ISSUE TITLE
"Both opening and closing quote characters must be on their own line."

### ISSUE THREAD

**Issue by phillijw** [dotnet/docs#41767]
### Type of issue

Other (describe below)

### Description

Misleading text. I would rephrase it as:

"Both opening and closing quote characters must be on separate lines."

At least I think that's how it's supposed to work.

### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/tokens/raw-string.md

### Document Version Independent Id

5737c986-9b06-9444-f7bc-893607d390d7

### Article author

@BillWagner

### Metadata

* ID: 62bd57d5-0e7d-4355-0689-4b6b90ab4057 
* Service: **dotnet-csharp**
* Sub-service: **lang-reference**

**Comment by BillWagner** [comment]
Hi @phillijw 

There's probably room for improvement here. I'd like to suggest alternatives to your proposal because I think that could be misinterpreted as well. Reading your text, I would expect the following to be legal (it's not):

```csharp
var rawString = """This is a
    multiline raw string literal. """;
```

The `"""` token is on "separate lines".

Maybe something like:

"In a multiline raw string literal, the opening quotes must be the last non-comment token on that line, and the closing quote must be first non-comment token on that line".

That's a bit pedantic, but it's more accurate.

**Comment by phillijw** [comment]
I wasn't actually 100% sure how it was supposed to work (which is why I was reading the docs in the first place) but I think your adjustment makes sense.
